theme/padplus/classes/output/core/course_renderer.php

```diff
@@ -215,5 +224,193 @@ protected function coursecat_category(coursecat_helper $chelper, $coursecat, $de
                 array('class' => 'category-item'));
         return $content;
     }
+    /*** PADPLUS END */
 
+    /*** PADPLUS:
+     * Displays one course in the list of courses.
+     *
+     * This is a trimmed down version of the original coursecat_coursebox, since we need to display different info
+     * depending on whether it is a sequence or workshop.
+     *
+     * @param coursecat_helper $chelper various display options
+     * @param core_course_list_element|stdClass $course
+     * @param string $additionalclasses additional classes to add to the main <div> tag (usually
+     *    depend on the course position in list - first/last/even/odd)
+     * @return string
+     */
+    protected function coursecat_coursebox(coursecat_helper $chelper, $course, $additionalclasses = '') {
+        if (!isset($this->strings->summary)) {
+            $this->strings->summary = get_string('summary');
+        }
+        if ($chelper->get_show_courses() <= self::COURSECAT_SHOW_COURSES_COUNT) {
+            return '';
+        }
+        if ($course instanceof stdClass) {
+            $course = new core_course_list_element($course);
+        }
+        $content = '';
+        $context = context_course::instance($course->id);
+        $coursebox = true;
+        $classes = trim('coursebox clearfix '. $additionalclasses);
+        $instances = enrol_get_instances($course->id, false);
+
+        if ($chelper->get_show_courses() < self::COURSECAT_SHOW_COURSES_EXPANDED) {
+            $classes .= ' collapsed';
+        }
+
+        // Coursebox.
+        $content .= html_writer::start_tag('div', array(
+            'class' => $classes,
+            'data-courseid' => $course->id,
+            'data-type' => self::COURSECAT_TYPE_COURSE,
+        ));
+
+        /*** PADPLUS: course image & course/workshop info. */
+        // Coursebox image.
+        $content .= html_writer::start_tag('div', array('class' => 'coursecat-image'));
+        $content .= $this->coursebox_image($course);
+        $content .= html_writer::end_tag('div');
+
+        // Check if it's a course or a workshop.
+        foreach ($instances as $key => $value) {
+            if ($value->enrol == 'self' && $value->status == '0') {
+                $coursebox = false;
+            }
+        }
+
+        // Coursebox info.
+        if ($coursebox) {
+            $content .= $this->coursebox_info($course, $chelper, $context, 'coursebox', null);
+        } else {
+            $rolelabel = get_string('workshop-teacher', 'theme_padplus');
+            $content .= $this->coursebox_info($course, $chelper, null, 'workshopbox', $rolelabel);
+        }
```

Comment:
if (workshop) {
  $rolelabel = get_string('workshop-teacher', 'theme_padplus'); // 
  $enrolledstudentsbox = ''; // do no show enrolled student number for workshop
  $buttonlabel = ;
  $boxclassname = ;
} else {
  $rolelabel = null; // Use user role as label
  $enrolledstudentsbox = $this->coursebox_enrolled_students($context);
  $buttonlabel = ;
  $boxclassname = ;   
}
$contactbox = $this->coursebox_contacts($course, $rolelabel);
$content .= $this->coursebox_info($course, $chelper, $contactbox, $enrolledstudentsbox, $buttonlabel, $boxclassname);